src/renault_api/cli/renault_vehicle.py

```diff
@@ -246,13 +246,19 @@ async def update_charge_mode(
     except KamereonResponseException as exc:  # pragma: no cover
         click.echo(f"charge-mode: {exc.error_details}", err=True)
     else:
-        items = [("Charge mode", response.chargeMode, None)]
+        if ctx_data["json"]:
+            status_table["charge-mode"] = response.raw_data
+            return
+        else:
```

Comment:
Can you remove the else statement and unindent below?
Since we now use return above it is not needed anymore.